ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -122,19 +122,22 @@ Item {
             section.property: "onlineStatus"
             section.delegate: (root.width > 58) ? sectionDelegateComponent : null
             delegate: StatusMemberListItem {
+                readonly property color userColor: Utils.colorForPubkey(model.pubKey)
+                readonly property var userColorHash: Utils.getColorHashAsJson(model.pubKey)
```

Comment:
:( I think this is a step backwards in a way. The model should provide all the data roles.

Reply:
I restored the previous behavior